### PR TITLE
Upgrade react-native to latest stable version and other npm dependencies - Closes #1083, #1085

### LINKED ISSUE #1083: Update npm dependencies
### Description

Update npm dependencies

### Motivation

The npm dependencies are outdated

```
found 555 vulnerabilities (8 low, 544 moderate, 2 high, 1 critical) in 2279 scanned packages
  run `npm audit fix` to fix 544 of them.
  11 vulnerabilities require manual review. See the full report for details.
```
### Additional Information

- Related https://github.com/LiskHQ/lisk-mobile/issues/1084

### PR DESCRIPTION
**Pull request by Balanced02**
### What was the problem?

This PR resolves #1083, #1085

### How was it solved?
- Upgraded react and react-native dependencies to latest
- Upgraded some other modules to latest

### Progress:
- Android build works fine, still working on iOS build

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,17 +23,15 @@ The next section details the prerequisites to install Lisk Mobile from source us
 
  - Git
    - `brew install git`
- - NodeJS (recommended 8.14.0)
-   - `brew install node`
- - npm (recommended 5.3.0)
+ - NodeJS (recommended 12.4.0)
+   - `brew install nvm`
```
Comment (ManuGowda) [inline README.md:27]:
```suggestion
   - `brew install node`
   - `brew install nvm`
```

File: README.md [hunk 2]
```diff
@@ -42,22 +40,69 @@ You'll need the Xcode installed in your machine to run the app in simulator.
...
+$ npm run android
 ```
 
 ### Development environment
 You can run the project in Xcode and use iOS simulators or alternatively use Android simulators. There are several options to set up your Android development environment. Please read [React Native docs](https://facebook.github.io/react-native/docs/getting-started.html) for more info.
 
-Three is a standalone app for debugging React Native. it has React and Redux debugger enabled by default. Please read  [React Native Debugger](https://github.com/jhen0409/react-native-debugger) for more info.
+There is a standalone app for debugging React Native. it has React and Redux debugger enabled by default. Please read  [React Native Debugger](https://github.com/jhen0409/react-native-debugger) for more info.
+
+### Possible Errors
```
Comment (ManuGowda) [inline README.md:72]: Since this mostly recommended approach for Mac users maybe add something like
Possible Errors (For Mac Users)?

Comment (Balanced02) [reply README.md:73]: Noted. Would update

**Review by ManuGowda — APPROVED** [review]

**Review by ManuGowda — COMMENTED** [review]
@Balanced02 since this setup only works for MacOS BigSur, should we mention the OS requirement?

**Comment by Balanced02** [comment]
> @Balanced02 since this setup only works for MacOS BigSur, should we mention the OS requirement?
@ManuGowda 
There's a line to use XCode's latest version, currently XCode 13 requires macOS Big Sur 11.3

File: README.md [hunk 3]
```diff
@@ -112,7 +160,7 @@ See [contributors section](https://github.com/LiskHQ/lisk-mobile/graphs/contribu
 
 ## License
 
-Copyright © 2016-2018 Lisk Foundation
+Copyright © 2016-2021 Lisk Foundation
```
Comment (ManuGowda) [inline README.md:163]:
```suggestion
Copyright © 2016-2018 Lisk Foundation
```

I had a conversation with Frederic (our legal department), for now lets keep it as it was, we will update once he gives us the direction to update.

**Review by ManuGowda — APPROVED** [review]

**Review by reyraa — APPROVED** [review]

**Pull request merged**